src/commands/schemas.ts

```diff
@@ -217,6 +222,107 @@ async function evolveSchemaSubjectCommand(subject: Subject) {
   await evolveSchemaCommand(schema);
 }
 
+/**
+ * Delete a single schema version.
+ *
+ * If was the last version bound to the subject, the subject will disappear also.
+ *
+ */
+async function deleteSchemaVersionCommand(schema: Schema) {
+  if (!(schema instanceof Schema)) {
+    logger.error("deleteSchemaVersionCommand called with invalid argument type", schema);
+    return;
+  }
+
+  const loader = ResourceLoader.getInstance(schema.connectionId);
+  const schemaGroup = await loader.getSchemasForSubject(schema.environmentId!, schema.subject);
+
+  // Ensure is still present in the registry / UI view gestured from wasn't stale.
+  const found = schemaGroup.find((s) => s.id === schema.id) !== undefined;
+  if (!found) {
+    vscode.window.showErrorMessage(`Schema not found in registry. View needs refresh?`);
```

Comment:
Could use a button here instead of the `View needs refresh?` to invite the user to force the refresh (and optionally include the "Open Logs"/"File Issue" buttons):
```suggestion
    showErrorNotificationWithButtons(
        "Schema not found in registry.",
        { "Refresh Schemas": () => commands.executeCommand("confluent.schemas.refresh"), ...DEFAULT_ERROR_NOTIFICATION_BUTTONS }    
    );
```

Reply:
Done, and expanded the coverage of when to show this to also handle case when entire subject is no longer findable (i.e. they deleted the last schema version of the subject in another window).